### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by notwaldorf**
Adds some nicer docs!

### CODE REVIEW

File: paper-fab.html [hunk 1]
```diff
@@ -165,7 +149,32 @@
...
+      }
+    },
+
+    observers: [
+      `_focusedChanged(focused)`
+    ],
+
+    ready: function() {
+      if (!this.hasAttribute('role')) {
+        this.setAttribute('role', 'button');
```
Comment (cdata) [inline paper-fab.html]: This should really be set in `hostAttributes`.

File: paper-fab.html [hunk 2]
```diff
@@ -165,7 +149,32 @@
...
       }
+      this._focusedChanged();
```
Comment (cdata) [inline paper-fab.html]: Why is this necessary? `focused` should have a default value?

File: paper-fab.html (hunk 1 above)
Comment (notwaldorf) [inline paper-fab.html]: Done

File: paper-fab.html (hunk 2 above)
Comment (notwaldorf) [inline paper-fab.html]: This is to display the initial elevation, from `raised`. `paper-button`, for example has `onRaisedChanged` but I didn't think it was needed here because `raised` is read only and can't actually change...

**Comment by notwaldorf** [comment]
Also, unrelated to this PR, all the tests are sad :(

**Comment by cdata** [comment]
Please do not merge this PR.

**Comment by notwaldorf** [comment]
Yup, won't.

File: paper-fab.html [hunk 3]
```diff
@@ -165,7 +154,30 @@
       mini: {
         type: Boolean,
         value: false
+      },
+
+      /**
+       * If true, the button should be styled with a shadow.
+       */
+      raised: {
```
Comment (cdata) [inline paper-fab.html]: Paper-fabs cannot be `raised`.

File: paper-fab.html [hunk 4]
```diff
@@ -165,7 +154,30 @@
...
+      `_focusedChanged(focused)`
+    ],
+
+    ready: function() {
+      // Show the initial elevation.
+      this._focusedChanged();
+    },
+
+    _focusedChanged: function(focused) {
+      this._calculateElevation();
```
Comment (cdata) [inline paper-fab.html]: This should already be called when `focused` changes.

File: paper-fab.html (hunk 3 above)
Comment (notwaldorf) [inline paper-fab.html]: I thought they were raised (i.e. not flat) by default. No? Does it have a different meaning?
